plugins/faustwp/includes/replacement/callbacks.php

```diff
@@ -200,13 +200,39 @@ function post_preview_link( $link, $post ) {
  * @return string URL used for the post.
  */
```

Comment:
Since we are preventing storing the Headless url in the database I think we need to add the following filter in line 24 for the `content_blocks` plugin:

```php
add_filter( 'wpgraphql_content_blocks_resolver_content', __NAMESPACE__ . '\\content_replacement');
```
This is to ensure that we trigger `domain replacement ` when resolving the blocks using `parse_blocks`.

Unless there is a better way... @josephfusco any hints?